iOSNativeSwiftTemplate/iOSNativeSwiftTemplate.xcodeproj/project.pbxproj

```diff
@@ -144,7 +144,7 @@
 			attributes = {
 				LastSwiftUpdateCheck = 0710;
 				LastUpgradeCheck = 0940;
-				ORGANIZATIONNAME = Salesforce;
+				ORGANIZATIONNAME = iOSNativeSwiftTemplateOrganizationName;
```

Comment:
@wmathurin it looked like this one didn't match up to its template.js, let me know if I misunderstood!

Reply:
Good catch.